### ISSUE TITLE
Adding chemistry components (both aerosols and gas species) into the regional_workflow

### ISSUE THREAD

**Issue by JianpingHuang-NOAA** [NOAA-EMC/regional_workflow#218]
The goal of this task is to bring the CAM-CMAQ or EMC_FV3GFS-AQM system into the regional workflow. The specific tasks include 

1) to bring in the AQM or CMAQ parts into the regional workflow;
2) to bing in the source code or scripts for  anthropogenic and fire emissions;
3) to bring in the codes/scripts for initial conditions and lateral boundary conditions;
4) to bring in data assimilation system (could be another issue for Cory later)
5) to be able to run the CAM/CMAQ
6) 5) to add the post-processing parts for chemical spices.

**Comment by jwolff-ncar** [comment]
@JianpingHuang-NOAA Would you like this issue to remain open?

**Comment by JeffBeck-NOAA** [comment]
I believe there is a separate CMAQ workflow being developed in parallel at the moment.  We need to decide whether that will remain separate or not.

**Comment by JacobCarley-NOAA** [comment]
@JeffBeck-NOAA this is the same task that @chan-hoo is now working on.

**Comment by JeffBeck-NOAA** [comment]
@JacobCarley-NOAA, thanks!  Is the eventual goal to merge the two workflows?

**Comment by JacobCarley-NOAA** [comment]
@JeffBeck-NOAA yes, that's the idea to eventually have these things together. The AQ bit is planned to be a part of RRFSv2, so it certainly makes sense.

**Comment by chan-hoo** [comment]
Until RRFSv2, refer to https://github.com/chan-hoo/regional_workflow/tree/rrfs_cmaq
